perun-base/src/main/resources/perun-roles.yml

```diff
@@ -219,40 +219,47 @@ perun_policies:
     include_policies:
       - default_policy
 
-  setAttribute_String_Attribute_policy:
-    policy_roles: []
-    include_policies:
-      - default_policy
-
   createAttribute_AttributeDefinition_policy:
     policy_roles: []
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA:
 
   deleteAttribute_AttributeDefinition_policy:
     policy_roles: []
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA:
 
   deleteAttribute_AttributeDefinition_boolean:
     policy_roles: []
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA:
 
   updateAttributeDefinition_AttributeDefinition_policy:
     policy_roles: []
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA:
 
   doTheMagic_Member_policy:
     policy_roles: []
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Member
```

Comment:
We said that setting attributes would be separately handled, and doTheMagic only sets attributes, if I understand correctly. Why 'MFA: Member' then? We don't do any changes to the Member object, or?

Reply:
in this case we're currently not even checking if the principal has the right to write to the attributes, maybe it is because only perunadmin is allowed to called the method. so i would stay consistent with it, and as user and member attributes can be changed, i'd keep MFA: Member, but add also MFA: User which i missed before.